substrate-archive/Cargo.toml

```diff
@@ -31,7 +31,7 @@ sqlx = { version = "0.5", default-features = false, features = ["postgres", "mac
 tempfile = "3.2.0"
 thiserror = "1.0"
 tracing = "0.1.26"
-tracing-subscriber = "0.2.15"
+tracing-subscriber = "=0.2.17"
```

Comment:
Why the `=`?

Reply:
CurrentSpan is being deprecated as per https://github.com/paritytech/substrate-archive/issues/272 , i pinned the version so that CI still passes

I'm removing CurrentSpan in #273